fabtests/multinode/src/harness.c

```diff
@@ -321,17 +321,20 @@ int main(int argc, char **argv)
 	if (!hints)
 		return EXIT_FAILURE;
 
-	while ((c = getopt(argc, argv, "n:C:z:Th" CS_OPTS INFO_OPTS)) != -1) {
+	while ((c = getopt(argc, argv, "n:x:z:Ths:" INFO_OPTS)) != -1) {
```

Comment:
Here a new option "-s" is added, but not used later.

Reply:
nvm, you are correct that -I isn't in INFO_OPTS. That also needs to be added manually.